src/main/webapp/app/exercises/shared/team/team-participation-table/team-participation-table.component.ts

```diff
@@ -134,6 +134,8 @@ export class TeamParticipationTableComponent implements OnInit {
      * @param submission Either submission or 'new'
      */
     getAssessmentLink(exercise: Exercise, participation: Participation, submission: Submission | 'new'): string[] {
```

Comment:
```suggestion
    getAssessmentLink(exercise: Exercise, participation: Participation | undefined, submission: Submission | undefined | 'new'): string[] {
```

Reply:
did that but with null since null is passed in the error case